Solr/5/templates/types.ss

```diff
@@ -367,7 +367,7 @@
 <fieldType name="lowercase" class="solr.TextField" positionIncrementGap="100">
     <analyzer>
         <tokenizer class="solr.KeywordTokenizerFactory"/>
-        <filter class="solr.LowerCaseFilterFactory"/>
+        <filter class="solr.`LowerCase`FilterFactory"/>
```

Comment:
Looks like stray ` ` `
```suggestion
        <filter class="solr.LowerCaseFilterFactory"/>
```

Reply:
Yaps, that is absolutely an accidental backtick on my end!